java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -3270,6 +3270,17 @@ public final Scalar getScalarElement(int index) {
     return new Scalar(getType(), getElement(getNativeView(), index));
   }
 
+  /**
+   * Get the number of bytes needed to allocate a validity buffer for the given number of rows.
+   * According to cudf::bitmask_allocation_size_bytes, the padding boundary for null mask is 64 bytes.
+   */
+  public static long getValidityBufferSize(int numRows) {
```

Comment:
nit: not public. The old API was package private

Reply:
Fixed.